### PR TITLE
Resolve minor Wording on Linked Posts

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#77502](http://central.tri.be/issues/77502)
---

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$this' | [Chooser_Meta_Box.php:152](https://github.com/moderntribe/the-events-calendar/blob/e86b037d44f16a1633e3f795961fe77eba5e528f/src/Tribe/Linked_Posts/Chooser_Meta_Box.php#L152)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$this' | [Chooser_Meta_Box.php:153](https://github.com/moderntribe/the-events-calendar/blob/e86b037d44f16a1633e3f795961fe77eba5e528f/src/Tribe/Linked_Posts/Chooser_Meta_Box.php#L153)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$this' | [Chooser_Meta_Box.php:152](https://github.com/moderntribe/the-events-calendar/blob/01b8920d5869b8745a3fc9f11656152d7560ce01/src/Tribe/Linked_Posts/Chooser_Meta_Box.php#L152)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$this' | [Chooser_Meta_Box.php:153](https://github.com/moderntribe/the-events-calendar/blob/01b8920d5869b8745a3fc9f11656152d7560ce01/src/Tribe/Linked_Posts/Chooser_Meta_Box.php#L153)

> via jenkins/codesniffer-phpcs

File: src/resources/js/events-admin.js [hunk 1]
```diff
@@ -326,8 +326,10 @@ jQuery( document ).ready( function( $ ) {
 				fields = $( create_template({}) );
 			}
 
+			console.log( fields );
+
 			// The final <tbody> contains the add new post link, we should add this new selector before that
```
Comment (borkweb) [inline src/resources/js/events-admin.js]: This comment should change.

**Comment by borkweb** [comment]
One small change is required (a comment). Once that is done, feel free to merge this. :+1:

**Pull request merged**